pkgs/tools/package-management/micromamba/default.nix

```diff
@@ -63,7 +63,6 @@ stdenv.mkDerivation rec {
     description = "Reimplementation of the conda package manager";
     homepage = "https://github.com/mamba-org/mamba";
     license = licenses.bsd3;
-    platforms = platforms.linux;
```

Comment:
it should be `platforms = platforms.all;`, otherwise neither hydra nor ofborg would build it

Reply:
TIL, thanks for the hint.